microdnf/commands/download/download.cpp

```diff
@@ -86,7 +86,12 @@ void CmdDownload::run(Context & ctx) {
         result_pset |= solv_query.get_package_set();
     }
 
-    download_packages(result_pset, ".");
+    std::vector<libdnf::rpm::Package> download_pkgs;
+    download_pkgs.reserve(result_pset.size());
+    for (auto package : result_pset) {
+        download_pkgs.push_back(std::move(package));
+    }
+    download_packages(download_pkgs, ".");
```

Comment:
Using for destdir value "." sounds strange to me.

Reply:
The download command downloads packages to current working directory.